src/index.js

```diff
@@ -16,8 +16,12 @@ import {
 
 const themed = key => props => props.theme[key]
 
+// shim for missing babel configs
+const css = props => props.css
```

Comment:
When the `css` prop is not enabled via Babel, this will still ensure that the `css` prop works on Rebass components